models/intermediate/int_amplitude__date_spine.sql

```diff
@@ -41,8 +54,8 @@ spine as (
     ) as spine
 
     {% if is_incremental() %} 
-    
-    where date_day > ( select max(date_day) from {{ this }} )
+        , max_date
+        where date_day >= max_date.max_event_day
```

Comment:
There isn't a need to create a buffer for the date spine (since the days won't change day over day). We should keep the same condition as before here. Otherwise, we may accidentally bring in duplicate days during incremental runs.
```suggestion
        where date_day > max_date.max_event_day
```